wasmtime-new-api/crates/wasi-common/build.rs

```diff
@@ -0,0 +1,20 @@
+// Tell any dependencies, if necessary, where our WASI submodule is so they can
+// use the same witx files if they want.
+fn main() {
+    let cwd = std::env::current_dir().unwrap();
```

Comment:
This should use the `CARGO_MANIFEST_DIR` env variable instead of pwd

Reply:
Fixed, thanks